shared/events.js

```diff
@@ -352,8 +364,8 @@ export class IdentityContract extends HashableContract {
   })
   static vuex = IdentityContract.Vuex({
     mutations: {
-      SetAttribute (state, {attribute: {name, value}}) { Vue.set(state.attributes, name, value) },
-      DeleteAttribute (state, {name}) { Vue.delete(state.attributes, name) }
+      SetAttribute (state, {data: {attribute: {name, value}}}) { Vue.set(state.attributes, name, value) },
+      DeleteAttribute (state, {data: {attribute: {name}}}) { Vue.delete(state.attributes, name) }
```

Comment:
The data contents is just `data.name`, not `data.attribute.name`, unless there's a good reason for that (e.g. multiple attribute deletion, in which case it'd be an array, not a map)